### PR TITLE
refactor: rename shadowed variables

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

File: src/nvim/window.c [hunk 1]
```diff
@@ -855,15 +855,15 @@ void ui_ext_win_position(win_T *wp)
       bool south = c.anchor & kFloatAnchorSouth;
 
       int comp_row = (int)row - (south ? wp->w_height_outer : 0);
-      int comp_col = (int)col - (east ? wp->w_width_outer : 0);
+      int _comp_col = (int)col - (east ? wp->w_width_outer : 0);
```
Comment (zeertzjq) [inline src/nvim/window.c:858]: This looks a bit inconsistent.

**Comment by zeertzjq** [comment]
Tbh I don't think this is good. Anyway I think it's better to makes the changes to `quickfix.c` and `syntax.c` to Vim first.

**Comment by dundargoc** [comment]
> Tbh I don't think this is good.

The underscore naming you mean? I can rename the variables to something else otherwise.

> Anyway I think it's better to makes the changes to quickfix.c and syntax.c to Vim first.

OK. Drafting this in the meantime.

**Comment by dundargoc** [comment]
There's no need for this PR if this is done with a vim-patch.

**Pull request closed without merging**